jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaHandlerResolver.kt

```diff
@@ -77,6 +77,14 @@ class JavaLambdaHandlerResolver : LambdaHandlerResolver {
      */
     override fun shouldShowLineMarker(handler: String): Boolean = !handler.contains("::")
 
+    /**
+     * Java Lambda handlers are equivalent if one is a subclass of RequestHandler and the other is the implementation method.
+     */
+    override fun areHandlersEquivalent(handler1: String?, handler2: String?): Boolean =
+        super.areHandlersEquivalent(handler1, handler2) ||
+            handler1 == "$handler2::handleRequest" ||
```

Comment:
Is it valid for the java method to be named anything other than `handlerRequest` ?

Reply:
Also is this logic tested in CreateLambdaFunctionTest.kt

Reply:
It is valid in Java world. The `handleRequest` is specifically used in the Lambda class interface. This is used to handle this edge case that:

1. Customers defined a Java class implementing `RequestHandler` in which it has a method `handleRequest`.
1. The gutter icon is only shown in the class level, not the method level since they are equivalent.
1. In the template, customer defines a handler with the method name. 
1. We still show the `Create Lambda Function` action in the class level gutter icon although there is no exact handler defined in the template.

Reply:
Added unit test for this method.